ebedke/plugins/podiumbistro.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+from io import BytesIO
+from PIL import Image
+import re
+
+from ebedke.utils.utils import get_fb_post_attached_image, on_workdays, ocr_image
+from ebedke.pluginmanager import EbedkePlugin
+
+
+FB_PAGE = "https://www.facebook.com/pg/PodiumBistroAndClub/posts/"
+FB_ID = "265810157626258"
+
+
+def menu_filter(post, today):
+    is_this_week = datetime.strptime(post['created_time'],
+                                     '%Y-%m-%dT%H:%M:%S%z').date() >= today.date() - timedelta(days=7)
+    text = post['message'].lower()
+    print(post['created_time'], post['message'])
```

Comment:
remove this please, it will show up in the logs :)

Reply:
Done.